src/components/Play.js

```diff
@@ -94,115 +93,55 @@ const PlayInfo = ({corpusId, playId}) => {
 
   const teiUrl = `${apiUrl}/corpora/${play.corpus}/play/${play.name}/tei`;
 
+  const castList = <CastList hasTitle cast={play.cast || []}/>;
+
   let tabContent = null;
+  let sidebar = null;
+
   if (tab === 'speech') {
-    tabContent = <SpeechDistribution segments={play.segments} {...{groups}}/>;
+    tabContent = (
+      <SpeechDistribution
+        type={chartType}
+        segments={play.segments}
+        {...{groups}}
+      />
+    );
+    sidebar = (
+      <SpeechDistributionNav
+        type={chartType}
+        onChange={type => setChartType(type)}
+      />
+    );
   } else if (tab === 'downloads') {
     tabContent = <DownloadLinks play={play}/>;
   } else if (tab === 'text') {
     tabContent = <TEIPanel url={teiUrl}/>;
+    sidebar = <SourceInfo source={play.source} original={play.originalSource}/>;
   } else if (tab === 'relations') {
     tabContent = <RelationsGraph {...{play, nodeColor, edgeColor}}/>;
+    sidebar = castList;
   } else {
     tabContent = <NetworkGraph {...{graph, nodeColor, edgeColor, play}}/>;
+    sidebar = castList;
```

Comment:
Would it also be possible to place network metrics into the network graph sidebar between the description and the cast list? Or do they have to remain inside the sigma container?

Reply:
Yes, I think we could move that box into the sidebar where you proposed. It could also gain a headline, like "Basic network properties".

Reply:
I also think we could do that. We should, however make sure there is enough space for all three of those sidebar items then. Maybe we could make the metrics collapsible so that the cast list could get more space if a user is more interested in the characters.

Reply:
Good point.